readme/apps/terminal.md

```diff
@@ -240,7 +241,8 @@ As an example, this is the default keymap, but read below for a detailed explana
 	{ "keys": ["mt"], "type": "prompt", "command": "mktodo \"\"", "cursorPosition": -2 },
 	{ "keys": ["mb"], "type": "prompt", "command": "mkbook \"\"", "cursorPosition": -2 },
 	{ "keys": ["yn"], "type": "prompt", "command": "cp $n \"\"", "cursorPosition": -2 },
-	{ "keys": ["dn"], "type": "prompt", "command": "mv $n \"\"", "cursorPosition": -2 }
+	{ "keys": ["dn"], "type": "prompt", "command": "mv $n \"\"", "cursorPosition": -2 },
+	{ "keys": ["z"], "command": "toggle_folder_collapse" }
```

Comment:
Same here. If we're going to update this we may as well update all that's missing